README.rst

```diff
@@ -941,6 +941,15 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_last`` (``bool``): whether ``null`` should be the last type; defaults to
```

Comment:
sorting alphabetically in descending order is weird... imo not needed

what's the difference between not setting up `null_position` and setting it with `default` value ?

`by_category` algorithm is a bit overkill imo, we should rather ask user to have one (maybe nullable) type instead of mixed types ;)

-------------

> Given this discussion, maybe it's better to implement whatever we agree on in this PR directly?

imo:
- sort_algorithm: 'alpha' (same, short value as in `ordered_imports` rule for consistency)
- nuill_position: 'always_first, always_last'

is enoguh for this PR. later, we could add other algorithms if needed